hollow/src/main/java/com/netflix/hollow/api/consumer/HollowConsumer.java

```diff
@@ -1212,6 +1214,31 @@ public B withLocalBlobStore(String localBlobStoreDir, boolean useExistingStaleSn
             return withLocalBlobStore(new File(localBlobStoreDir), useExistingStaleSnapshot);
         }
 
+        /**
+         * Provide a directory that will be used to cache blobs.
+         *
+         * This is the same as {@link #withLocalBlobStore(File, boolean)} but also allows
+         * specifying the maximum age of an existing local blob. When a local blob is found
+         * that is older than the specified maximum age, it will not be used and Hollow will
+         * delete it, falling back to retrieving the blob using the configured blobRetriever.
+         *
+         * @param existingBlobMaxAge the maximum age an existing local blob can be before
+         *   it is deleted and not used.
+         */
+        public B withLocalBlobStore(File localBlobStoreDir, boolean useExistingStaleSnapshot, Duration existingBlobMaxAge) {
```

Comment:
nit: maybe just the one that takes `File` seems sufficient. Not sure why we did both `File` and `String` variants earlier, guessing `File` came later

Reply:
Updated to just have the `File` constructor variant.